compact/far_forward_arches.xml

```diff
@@ -8,7 +8,7 @@
   <include ref="far_forward/B0_tracker.xml"/>
   <include ref="far_forward/B0_ECal.xml"/>
   <include ref="far_forward/offM_tracker.xml"/>
-  <include ref="far_forward/ZDC_Athena.xml"/>
+  <include ref="far_forward/ZDC.xml"/>
```

Comment:
We could consider moving this stuff to https://github.com/eic/epic/blob/main/templates/epic.xml.jinja2 and add some configutation conditionals there.

Reply:
I am not sure that is necessary - the "ZDC.xml" file is the baseline option.

Reply:
Okay.